pkg/log/main.go

```diff
@@ -11,27 +11,113 @@ import (
 	"github.com/decentralized-identity/kerigo/pkg/event"
 )
 
+type Escrow map[string]*event.Message
+
+// Get returns the event message with all collected signatures
+// for the given event
+func (e Escrow) Get(evnt *event.Event) (*event.Message, error) {
+	serialized, err := evnt.Serialize()
+	if err != nil {
+		return nil, err
+	}
+
+	digest, err := event.DigestString(serialized, derivation.Blake3256)
+	if err != nil {
+		return nil, err
+	}
+
+	escrowed := &event.Message{}
+
```

Comment:
Do you want to fill in the Event that was passed in here?

Reply:
Yep, good catch